com.unity.netcode.gameobjects/Prototyping/NetworkTransform.cs

```diff
@@ -9,19 +9,7 @@ namespace Unity.Netcode.Prototyping
     [AddComponentMenu("Netcode/" + nameof(NetworkTransform))]
     public class NetworkTransform : NetworkBehaviour
     {
-        /// <summary>
-        /// Server authority allows only the server to update this transform
-        /// Client authority allows only the owner client to update this transform
-        /// Shared authority allows everyone to update this transform
-        /// </summary>
-        public enum NetworkAuthority
-        {
-            Server = 0,
-            Client,
-            Shared
-        }
-
-        internal class NetworkState : INetworkSerializable
+        internal struct NetworkState : INetworkSerializable
```

Comment:
obv, this is more than just "remove client network permissions", isn't it?

should we have perms changes under a separate PR and come back to this class to struct conversion later?
I'm also working on this code, I might just task myself up for class to struct conversion if you'd like.